README.md

```diff
@@ -1,34 +1,44 @@
-# GRASS Addons git repository
+# GRASS GIS Addons
 
-## How to get write access here
+This is the official GRASS GIS Addons git repository.
```

Comment:
"git repository" sounds like there is some other one. Well, maybe there is (binaries for Windows), but is that relevant here? In any case, Git should be capitalized (the [software uses first upper case](https://git-scm.com/about) when it is not the command line tool name).